### DISCUSSION TITLE
Is it possible to add 'fake' lenses to the media I send from the gallery?

### DISCUSSION THREAD

**Discussion by KimiRaikkonen11** [rodit/SnapMod#94] [Ideas & Feature Suggestions]
For example, I have old snaps saved in my camera roll that use a lens. Is it possible to send this media to others to indicate that you used that lens?

**Comment by rodit** [comment]
I could look into this - it might be quite an interesting feature to implement, although I imagine it will be quite difficult to allow the user to select a lens they want to 'fake' easily.

**Reply by jammyjay** [reply]
Wondering if any update on this? The ability to show a lens was used for older saved snaps

**Reply by rodit** [reply]
I haven't got round to this yet unfortunately - I'll probably have a look sometime next week.

**Comment by RevealedSoulEven** [comment]
@rodit Better add an option to add link of the lenses
And you can convert that link to the lense ID because I guess snapchat API uses lense ID or something for that lense part to show in the snaps
